src/main/java/betterquesting/handlers/EventHandler.java

```diff
@@ -91,7 +92,10 @@ public void onKey(InputEvent.KeyInputEvent event) {
             if (BQ_Settings.useBookmark && GuiHome.bookmark != null) {
                 mc.displayGuiScreen(GuiHome.bookmark);
             } else {
-                mc.displayGuiScreen(ThemeRegistry.INSTANCE.getGui(PresetGUIs.HOME, GArgsNone.NONE));
+                GuiScreen guiToDisplay = ThemeRegistry.INSTANCE.getGui(PresetGUIs.HOME, GArgsNone.NONE);
+                if (BQ_Settings.useBookmark && BQ_Settings.skipHome)
```

Comment:
This probably does not need to check `useBookmark`, as skipping the home page should be separate from jumping to the last quest.